packages/styles/vite.config.js

```diff
@@ -11,6 +11,7 @@ export default defineConfig({
   },
   plugins: [],
   build: {
+    minify: false,
```

Comment:
Why not minify? The browser and VSCode can both reformat it as readable. Does this affect something else?

Reply:
Didn't even think about that haha